.vscode/launch.json

```diff
@@ -26,51 +26,21 @@
       "webRoot": "${workspaceFolder}/app/javascript/packs"
     },
     {
-      "name": "Debug Local File",
```

Comment:
I ended up omitting this task rather than trying to replace it since
- it looks like it probably wasn't getting used (there is no longer a `main.rb` file at the repo root)
- i couldn't think of a use case where executing a solo `.rb` file in a Rails app would be needed / helpful for debugging